src/components/dictionaryConcepts/components/ExternalMapping.jsx

```diff
@@ -1,93 +1,84 @@
-                <option>Narrower than</option>
-              </select>
-            </FormGroup>
-          </div>
-          <div>
-            <FormGroup className="form-style">
-              Concept
-              <Input
-                type="text"
-                className="form-control answer"
-                placeholder="eg. /orgs/Regenstrief/sources/loinc2/concepts/32700-7/"
-                name="source_url"
-                value={source_url}
-                onChange={this.handleChange}
-                required
-              />
-            </FormGroup>
-          </div>
-          <div>
-            <FormGroup className="form-style">
-              Code
-              <Input
-                type="text"
-                className="form-control answer"
-                placeholder="eg. jgug-fgtgtg-rfrg"
-                name="code"
-                value={code}
-                onChange={this.handleChange}
-                required
-              />
-            </FormGroup>
-          </div>
-          <div>
-            <FormGroup className="form-style">
-              Term
-              <Input
-                type="text"
-                className="form-control answer"
-                placeholder="eg. Malaria"
-                name="term"
-                value={term}
-                onChange={this.handleChange}
-                required
-              />
-            </FormGroup>
-          </div>
-        </form>
-      </div>
-    );
-  }
-}
+  return (
+    <Fragment>
+      <FormGroup className="form-style">
+        Mapping source concept
+        <Input type="text" value={concept.display_name} readOnly />
+      </FormGroup>
+      <FormGroup className="form-style">
+        Concept map type
+        <b className="text-muted">&nbsp;*</b>
+        <Input
+          type="text"
+          className="form-control answer"
+          placeholder="Concept map type ex: Same"
+          name="mapType"
+          value={mapType}
+          onChange={handleChange}
+          autoComplete="off"
+          id="mapType"
+          required
+        />
+        <small className="form-text text-danger">{errors.mapType}</small>
+      </FormGroup>
+      <FormGroup className="form-style">
+        Destination dictionary
+        <AsyncSelect
+          name="toDictionaryUrl"
+          placeholder="Mapping destination dictionary ..."
+          valueKey="url"
+          labelKey="name"
+          sourceUrl={dictionarySearchUrl}
+          ignoreValue={dictionarySearchUrl}
+          onChange={handleAsyncSelect}
+        />
+        <small className="form-text text-danger">{errors.toDictionaryUrl}</small>
+      </FormGroup>
+      <FormGroup className="form-style">
+        Destination concept
+        <AsyncSelect
+          name="toConceptUrl"
+          placeholder="Mapping destination concept ..."
+          valueKey="id"
+          labelKey="display_name"
+          sourceUrl={conceptSearchUrl}
```

Comment:
Thanks for the effort @muhozi . You can add some spaces to contents in braces.

Reply:
@collinewait I don't think that there should not be a space according to [this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md)